packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -1,14 +1,14 @@
 <template>
   <div id="oc-file-details-sidebar">
     <oc-loader v-if="loading" />
-    <div v-else>
+    <div v-if="!loading && hasContent">
```

Comment:
```suggestion
    <div v-else-if="hasContent">
```

Reply:
Adding this and bumping ODS to `v8.0.0` once it's released, then re-running CI and will merge this once it's green!